### PR TITLE
Make Carbon bot use the Climatiq API

### PR DESCRIPTION
**Pull request by samsucik**
The Climatiq API estimates CO2 given a distance or IATA codes. To make requests, we need an API key.

_An important part of using the API is [choosing the emission factors to be used](https://explorer.climatiq.io/?category=Air+Travel&sector=Transport). An emission factor is a coefficient used to compute the CO2 estimate from parameters like distance or flight class. Emission factors differ depending on the distance: E.g., long-haul flight planes fly higher, where the real impact of emitted CO2 is also higher. Different emission factors exist based on different methodologies, e.g. in the UK the emission factors also account for a plane's contrails, for the different greenhouse gases emitted, and upweight the estimate in a particular way for long-haul flights. Institutions in other countries produce different numbers as they use slightly different approaches._

In this PR, I chose the UK emission factors (but it's easy to switch to ones from a different source). They seem to be quite conservative, i.e. they account for emission effects that get ignored in many other emission factors. The UK factors also distinguish between domestic and international flights in a particular way that varies between countries and methodologies -- in this case, `domestic` means any flight within the UK, i.e. up to ~760 km.

Besides choosing the emission factors, this PR also adds the API key into the action server Docker image so that requests can be made when the action server is deployed (the actual API key is stored as a secret on the `carbon-bot` repo).

In terms of testing that everything works (around the API), I've checked this locally but I admit that we might want to [set up a unit test for the action server](https://forum.rasa.com/t/how-do-i-test-my-custom-actions/39020).

### CODE REVIEW

File: actions.py [hunk 1]
```diff
@@ -121,6 +137,36 @@ def co2_kg_interpolation(distance_km, flight_class, include_rfi=True):
 
         return co2_kg_per_km * distance_km
 
+    @staticmethod
+    def co2_kg_climatiq(distance_km, flight_class, departure_iata, destination_iata):
+        if distance_km < 760:  # longest UK domestic flight (Inverness<->Gatwick)
+            emission_factor = CLIMATIQ_EMISSION_FACTORS["domestic"]
+        elif (
+            distance_km < 3700
```
Comment (tttthomasssss) [inline actions.py]: Should we have these as constants somewhere?

File: actions.py [hunk 2]
```diff
@@ -169,6 +215,11 @@ def calculate_emissions(self, tracker, unit="short_tons"):
             raise ValueError("Destination IATA code unknown.")
         if not flight_class:
             raise ValueError("Flight class unknown.")
```
Comment (tttthomasssss) [inline actions.py]: Don't really need this conditional given the one afterwards (which also has a nicer error message)

**Review by tttthomasssss — APPROVED** [review]
All LGTM 🚀 - awesome work!

Consider the comments optional.

File: actions.py (hunk 1 above)
Comment (tttthomasssss) [reply actions.py]: (as in, the two distance thresholds)

Comment (samsucik) [reply actions.py]: Yeah, that'd be better. Thanks for the suggestion 🙂

File: actions.py (hunk 2 above)
Comment (samsucik) [reply actions.py]: True 🙂

**Review by tczekajlo — APPROVED** [review]
LGTM

**Pull request merged**